otpset/README.md

```diff
@@ -0,0 +1,8 @@
+# otpset
+Wrapper script for programming the OTP on Raspberry Pi devices.
+
+OTP setting is done via `vcmailbox` calls, and are not particularly easy to use. This wrapper makes it easier to read and set customer OTP bits, and provides some level of error checking to the process.
+
+Once OTP bits are set, the CANNOT be unset, so this wrapper will ensure the user is sure when setting bits. 
```

Comment:
"the CANNOT" -> "they CANNOT"
"will ensure the user is sure when setting bits" -> "requests confirmation before performing destructive/irreversible operations."